File: Scadex.Model/Dtos/ComponentTemplate/Queries/TemplateImageDto.cs
using Scadex.Core.Model;

namespace Scadex.Model.Dtos.ComponentTemplate.Queries;

/// <summary>Yuklenen ComponentTemplate gorselinin sonucu. </summary>
public class TemplateImageDto : IDto
{
    public string Url { get; set; } = null!;
}

File: Scadex.WebUI/src/models/componentTemplate/queries/templateImageDto.ts
/** Ayna: Scadex.Model/Dtos/ComponentTemplate/Queries/TemplateImageDto.cs — sözleşme: docs/api-contract/10-component-template.md */

/**
 * Yüklenen şablon görselinin sonucu.
 *
 * `url` GÖRELİDİR (`/uploads/templates/x.png`). Mutlak adres saklamak, uygulama
 * taşındığı gün (localhost → IIS → ters vekil) bütün şablonların görselini
 * kırardı. Tarayıcıya vermeden önce `templateImageSrc()` ile API adresiyle
 * birleştirilir.
 */
export interface TemplateImageDto {
  url: string;
}
